<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="tr_TR">
<context>
    <name>QObject</name>
    <message>
        <source>CONNECTION ERROR</source>
        <translation>BAĞLANTI HATASI</translation>
    </message>
    <message>
        <source>SQL QUERY ERROR</source>
        <translation>SQL SORGU HATASI</translation>
    </message>
    <message>
        <source>EXEC DDL ERROR</source>
        <translation>EXEC DDL HATASI</translation>
    </message>
    <message>
        <source>UNKNOWN DRIVER</source>
        <translation>TANIMSIZ SÜRÜCÜ</translation>
    </message>
    <message>
        <source>Used Database           :</source>
        <translation>Kullanılan Veritabanı:</translation>
    </message>
    <message>
        <source>Server                  :</source>
        <translation>Sunucu                :</translation>
    </message>
    <message>
        <source>SQL Engine              :</source>
        <translation>SQL Motoru            :</translation>
    </message>
    <message>
        <source>Operating System        :</source>
        <translation>İşletim Sistemi             :</translation>
    </message>
    <message>
        <source>Error Message           :</source>
        <translation>Hata Mesajı                :</translation>
    </message>
    <message>
        <source>Error Type              :</source>
        <translation>Hata Türü              :</translation>
    </message>
    <message>
        <source>Error No                :</source>
        <translation>Hata no                :</translation>
    </message>
    <message>
        <source>Comment                 :</source>
        <translation>Açıklama                 :</translation>
    </message>
    <message>
        <source>Incorrect Query         :</source>
        <translation>Hatalı Sorgu              :</translation>
    </message>
    <message>
        <source>Error Date              :</source>
        <translation>Hata Tarihi            :</translation>
    </message>
    <message>
        <source>Error Time              :</source>
        <translation>Hata Saati               :</translation>
    </message>
    <message>
        <source>Database connection failed.</source>
        <translation>Veritabanı bağlantısı başarısız.</translation>
    </message>
    <message>
        <source>An error was returned from the ADAK_SQL function.</source>
        <translation>ADAK_SQL kurucu fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>An error was returned from the CREATE_DATABASE function.</source>
        <translation>CREATE_DATABASE kurucu fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>An error was returned from the CREATE_TABLE function.</source>
        <translation>CREATE_TABLE fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>An error was returned from the DROP_DATABASE function.</source>
        <translation>DROP_DATABASE fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>An error was returned from the DROP_TABLES function.</source>
        <translation>DROP_TABLE fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>An error was returned from the USER_EXISTS function.</source>
        <translation>USER_EXISTS fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>An error was returned from the CREATE_USER function.</source>
        <translation>CREATE_USER fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>An error was returned from the unknown function.</source>
        <translation>Bilinmeyen bir fonksiyondan hata döndürüldü.</translation>
    </message>
    <message>
        <source>An error was returned from the CREATE_DATABASE function. </source>
        <translation>CREATE_DATABASE fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>TRANSACTION to start desired when TRANSACTION opened. Intertwined TRANSACTION&apos;s can not be opened.</source>
        <translation>TRANSACTION açıkken yeni TRANSACTION başlatılmak istendi. İç içe TRANSACTION &apos; lar olamaz.</translation>
    </message>
    <message>
        <source>COMMIT_TRANSACTION was called without TRANSACTION.</source>
        <translation>TRANSACTION yokken COMMIT_TRANSACTION çağrıldı.</translation>
    </message>
    <message>
        <source>CANCEL_TRANSACTION was called without TRANSACTION.</source>
        <translation>TRANSACTION yokken CANCEL_TRANSACTION çağrıldı.</translation>
    </message>
    <message>
        <source>Meantime SELECT, you can use PREPARE_LIMIT SELECT for LIMIT.</source>
        <translation>SELECT esnasinda LIMIT icin PREPARE_LIMIT_SELECT kullanmalisiniz.</translation>
    </message>
    <message>
        <source>An error was returned from the UPDATE() function.</source>
        <translation>UPDATE() fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>An error was returned from the DELETE()function.</source>
        <translation>DELETE() fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>INSERT FUNCTION ERROR 
</source>
        <translation>INSERT FONKSİYONU HATASI</translation>
    </message>
    <message>
        <source>UPDATE FUNCTION ERROR 
</source>
        <translation>UPDATE FONKSİYONU HATASI</translation>
    </message>
    <message>
        <source>DELETE FUNCTION ERROR 
</source>
        <translation>DELETE FONKSİYONU HATASI</translation>
    </message>
    <message>
        <source>SELECT FUNCTION ERROR 
</source>
        <translation>SELECT FONKSİYONU HATASI</translation>
    </message>
    <message>
        <source>The table name not found.</source>
        <translation>Tablo bulunamadı.</translation>
    </message>
    <message>
        <source>An error was returned from FIND_TABLE_ID function.</source>
        <translation>FIND_TABLE_ID() fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>VALUE() Error. Query NEXT() forgotten or x is wrong in VALUE(x).</source>
        <translation>VALUE()&apos;da hata. Ya NEXT() unutulmus ya da VALUE(x) teki x yanlis.</translation>
    </message>
    <message>
        <source>SET_VALUE has been used to unnecessary or incorrect domain name.</source>
        <translation>SET_VALUE gereksiz veya yanlis alan adi ile kullanilmis.</translation>
    </message>
    <message>
        <source>To assign SET_VALUE forgotten.</source>
        <translation>SET_VALUE atamasi unutulmus.</translation>
    </message>
    <message>
        <source>UNKNOWN ERROR TYPE.</source>
        <translation>BİLİNMEYEN HATA TÜRÜ.</translation>
    </message>
    <message>
        <source>An error was returned from the START_TRANSACTION() function.</source>
        <translation>START_TRANSACTION() fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>An error was returned from the COMMIT_TRANSACTION() function.</source>
        <translation>COMMIT_TRANSACTION() fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>An error was returned from the CANCEL_TRANSACTION() function.</source>
        <translation>CANCEL_TRANSACTION() fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>An error was returned from the PREPARE_SELECT() function.</source>
        <translation>PREPARE_SELECT() fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>An error was returned from the PREPARE_INSERT() function.</source>
        <translation>PREPARE_INSERT() fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>An error was returned from the PREPARE_UPDATE() function.</source>
        <translation>PREPARE_UPDATE() fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>An error was returned from the PREPARE_DELETE() function.</source>
        <translation>PREPARE_DELETE() fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>An error was returned from the INSERT() function.</source>
        <translation>INSERT() fonksiyonundan hata döndürüldü.</translation>
    </message>
    <message>
        <source>ERROR TYPE IS NOT DETECTED.</source>
        <translation>HATA TÜRÜ BULUNAMADI.</translation>
    </message>
    <message>
        <source>This operation can not be performed before the transaction starts.</source>
        <translation>Transaction başlatılmadan bu işlem gerçekleştirilemez.</translation>
    </message>
    <message>
        <source>ADAK_SQL does not support this SQL engine.
</source>
        <translation>ADAK_SQL belirttiginiz SQL motorunu desteklemiyor.</translation>
    </message>
    <message>
        <source>Supported: PostgreSQL, MySQL, SQLite, Microsoft SQL , ORACLE.</source>
        <translation>Desteklenenler: PostgreSQL, MySQL, SQLite, Microsoft SQL , ORACLE.</translation>
    </message>
    <message>
        <source>The column name found in SET_VALUE function.</source>
        <translation>SET_VALUE() fonksiyonunda belirtilen kolon adı bulunamadı.</translation>
    </message>
    <message>
        <source>Query is not SELECT(), or SELECT () is called before the NEXT () was called.</source>
        <translation>Ya sorgu SELECT degil, yada SELECT() cagrilmadan NEXT() cagrildi.</translation>
    </message>
</context>
<context>
    <name>tQObject</name>
    <message>
        <source>CREATION ERROR</source>
        <translation>OLUŞTURMA HATASI</translation>
    </message>
</context>
</TS>
